content/applications/hr/employees/new_employee.rst

```diff
@@ -210,57 +266,73 @@ box, or typing in the information.
+- :guilabel:`Dependant`: If the employee has any children, enter the :guilabel:`Number of Children`
+  in the field.
 - :guilabel:`Work Permit`: If the employee has a work permit, enter the information in this section.
   Type in the :guilabel:`Visa No` and/or :guilabel:`Work Permit No` in the corresponding fields.
-  Using the calendar module, select the :guilabel:`Visa Expire Date` to enter the expiration date.
+  Using the calendar module, select the :guilabel:`Visa Expire Date` and/or the :guilabel:`Work
+  Permit Expiration Date` to enter the expiration date(s). If available, upload a digital copy of
+  the work permit document. Click :guilabel:`Upload Your File`, navigate to the work permit file in
+  the file explorer, and click :guilabel:`Open`.
 
 .. image:: new_employee/private-info.png
    :align: center
    :alt: Add the private information to the Private Information tab.
 
-HR settings tab
+HR Settings tab
 ---------------
 
 This tab provides various fields for different information, depending on the country the company is
 located. Different fields are configured for different locations, however some sections appear
 regardless.
 
-- :guilabel:`Status`: If applicable, select a :guilabel:`Related User`, :guilabel:`Job Position`,
-  and :guilabel:`Language` with the drop-down menus. Type in the :guilabel:`Registration Number of
-  the Employee`, and the :guilabel:`NIF Country Code` if available.
-- :guilabel:`Fleet`: If the employee has access to a company car, enter the :guilabel:`Mobility
-  Card` information here.
-- :guilabel:`Timesheets`: Enter the employee's cost per hour in a $XX.XX format. This is factored in
-  when the employee is working at a work center. This value affects the employee's pay, and may also
-  affect manufacturing costs for a product, if the value of the manufactured product is not a fixed
-  amount.
-- :guilabel:`Attendance`: This section may only appear for Belgian companies. Enter the
-  :guilabel:`INSZ or BIS` number if applicable. :guilabel:`Badge ID` and a :guilabel:`PIN Code` can
-  be entered here, if the employee needs/has one. Click :guilabel:`Generate` next to the
+- :guilabel:`Status`: Select an :guilabel:`Employee Type` and, if applicable, a  :ref:`Related User
+  <employees/related-user>`, with the drop-down menus.
+- :guilabel:`Payroll`: Select the :guilabel:`Current Contract` and :guilabel:`Job Position` from the
+  drop-down menus. If applicable, enter the :guilabel:`Registration Number` in this section.
+- :guilabel:`Previous Employer`: This section appears only for Belgian companies, and will not be
+  visible for other locations. These are days that will be paid to the new employee. Enter any
+  :guilabel:`Simple Holiday Pay to Recover`, :guilabel:`Number of Days to recover`, and
+  :guilabel:`Recovered Simple Holiday Pay` from a previous employer, for both N and N-1 categories.
+- :guilabel:`Previous Occupations`: This section appears ony for Belgian companies, and will not be
```

Comment:
I have this at the top : This tab provides various fields for different information, depending on the country the company is
located. Different fields are configured for different locations, however some sections appear
regardless.
I think that might suffice since it was in the 14 and 16 docs, too.